integration-tests/cheqd-credentials.test.ts

```diff
@@ -0,0 +1,59 @@
+import { IWallet } from '@docknetwork/wallet-sdk-core/lib/types';
+import { createVerificationController } from '@docknetwork/wallet-sdk-core/src/verification-controller';
+import { CheqdCredentialNonZKP } from './data/credentials/cheqd-credentials';
+import { closeWallet, createNewWallet, getCredentialProvider, getWallet } from './helpers';
+import { ProofTemplateIds, createProofRequest } from './helpers/certs-helpers';
+
+describe('Cheq integration tests', () => {
+  beforeAll(async () => {
+    await createNewWallet();
+  });
+
+  it('should verify a non ZKP cheqd credential', async () => {
```

Comment:
do we need one for ZKP too?

Reply:
yes, would be good to add one here. I was planning to add zkp with accumulators, but makes sense to have zkp without accumulators too

Reply:
We are facing few issues with zkp verification, I will push that test as separate PR probably along with accumulators tests